specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -0,0 +1,604 @@
+					}
+				},
+				"tags": ["SubAssessments"],
+				"description": "Get a security sub-assessment on your scanned resource",
+				"operationId": "SubAssessments_Get",
+				"parameters": [{
+					"$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+				},
+				{
+					"$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+				},
+				{
+					"$ref": "#/parameters/AssessmentName"
+				},
+				{
+					"$ref": "#/parameters/SubAssessmentName"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/SecuritySubAssessment"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "../../../common/v1/types.json#/definitions/CloudError"
+						}
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"SecuritySubAssessmentList": {
+			"type": "object",
+			"description": "List of security sub-assessments",
+			"properties": {
+				"value": {
+					"readOnly": true,
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/SecuritySubAssessment"
+					}
+				},
+				"nextLink": {
+					"readOnly": true,
+					"type": "string",
+					"description": "The URI to fetch the next page."
+				}
+			}
+		},
+		"SecuritySubAssessment": {
+			"type": "object",
+			"description": "Security sub-assessment on a resource",
+			"properties": {
+				"properties": {
+					"x-ms-client-flatten": true,
+					"$ref": "#/definitions/SecuritySubAssessmentProperties"
+				}
+			},
+			"allOf": [{
+				"$ref": "../../../common/v1/types.json#/definitions/Resource"
+			}]
+		},
+		"SecuritySubAssessmentProperties": {
+			"type": "object",
+			"description": "Describes properties of an sub-assessment.",
+			"properties": {
+				"id": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Vulnerability ID."
+				},
+				"displayName": {
+					"readOnly": true,
+					"type": "string",
+					"description": "User friendly display name of the sub-assessment"
+				},
+				"status": {
+					"$ref": "#/definitions/SubAssessmentStatus"
+					
+				},
+				"remediation": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Information on how to remediate this sub-assessment"
+				},
+				"impact": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Description of the impact of this sub-assessment"
+				},
+				"category": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Category of the sub-assessment"
+				},
+				"description": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Human readable description of the assessment status"
+				},
+				"timeGenerated": {
+					"readOnly": true,
+					"type": "string",
+					"format": "date-time",
+					"description": "The date and time the sub-assessment was generated"
+				},
+				"resourceDetails": {
```

Comment:
>resourceDetails [](start = 5, length = 15)

nit nit - assessedResource ?

Reply:
@orparnes added few comments

Reply:
> > resourceDetails [](start = 5, length = 15)
> 
> nit nit - assessedResource ?

We want to be aligned with "assessments" API, so prefer to leave it as-is